README.md

```diff
@@ -1,106 +1,275 @@
+const datafileURL = 'https://cdn.optimizely.com/datafiles/626QeFJNZRdW8SkvGSFEuq.json'; <- replace this with your URL
 
-### Deploying the App
-1. Login or create an [Optimizely Account](https://app.optimizely.com/signin).
-2. Create a project via the Optimizely dashboard. [Instructions](http://developers.optimizely.com/server/getting-started/index.html?language=javascript)
-3. Add a Feature with the key `sorting_enabled`. This will act as a toggle for our new feature. 
-4. Add the Event Key `item_purchase`.
-5. In `constants.js`, update the `datafileURL` field. This is found in the dashboard, under Settings -> Environments.
-6. Install dependencies & run the application
+module.exports = {
+  datafileURL
+}
+```
+8. Save the file. 
+
+### Running the Server and Demo
+
+1. Open a terminal window and navigate to the root of the **javascript-sdk-demo-app** package.
+2. Install the dependencies:
+```shell
+npm install
+```
+3. Run the server:
+```shell
+npm start
 ```
-$ npm install
-$ npm start
+Once the server is executing, the output should indicate that it's running on Port 8080.
+4. Open a browser and enter `http://localhost:8080` as the URL. The demo webpage should display.
+
+## How the Test App was Created
+The following subsections provide information about key aspects of the demo and how it was put together:
+* [Project Structure](#project-structure)
+* [SDK Packaging](#sdk-packaging)
+* [User Interface](#user-interface)
+* [Visual Assets](#user-interface-and-visual-assets)
+* [Styling](#styling)
+
+### Package Structure
+The following are the main components of interest in the package:
+
+1. **constants.js**: contains the URL of the datafile to use for the experiment. The datafile acts as a config file and represents the state of your Optimizely project. It contains information like the status of your features, experiments, configuration parameters, and traffic allocation.
+2. **webpack.config.js**: contains configuration information describing how the SDK is [bundled](#sdk-packacking) with the demo. 
+3. **server.js**: contains the node.js server to run the demo locally on Port 8080.
+4. **./node_modules**: contains the node.js dependencies.
+5. **./src**: contains the HTML, Javascript, and resources for the (client-side) demo webpage. **index.html** contains the base web page UI that is displayed. **purchase.html** contains a message indicating that an event is being tracked by Optimizely.
+
+### SDK Packaging
+Instead of including the [Optimizely Javascript SDK](https://github.com/optimizely/javascript-sdk) on the page as a standalone JavaScript asset, the SDK was bundled with the web site's resources using [webpack](https://webpack.js.org). webpack bundles the source code, dependencies, application and experimentation logic, and the Optimizely SDK into a single bundle and is configured using the webpack configuration file: **./webpack.config.js**:
```

Comment:
nit. JavaScript and not Javascript. Please make this change across all files in this PR.